pkgs/by-name/cu/cunicu/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  protobuf,
+  protoc-gen-go,
+  protoc-gen-go-grpc,
+}:
+buildGoModule rec {
+  pname = "cunicu";
+  version = "0.5.53";
+
+  src = fetchFromGitHub {
+    owner = "cunicu";
+    repo = "cunicu";
+    rev = "v${version}";
+    hash = "sha256-y8JyXf+LdquhVr9G5wX1LdMF7wpj5PZG3xXGAWoypzA=";
+  };
+
+  nativeBuildInputs = [
+    protobuf
+    protoc-gen-go
+    protoc-gen-go-grpc
+  ];
+
+  CGO_ENABLED = 0;
+
+  vendorHash = "sha256-OiLVdEf6fcGHx0k0xC5sZwhnK0FiLgfdkz2zNgBbcgY=";
+
+  # These packages contain networking dependent tests which fail in the sandbox
+  excludedPackages = [
+    "pkg/config"
+    "pkg/selfupdate"
+    "pkg/tty"
+    "scripts"
+  ];
+
+  ldflags = [
+    "-X"
+    "cunicu.li/cunicu/pkg/buildinfo.Version=${version}"
+    "-X"
+    "cunicu.li/cunicu/pkg/buildinfo.BuiltBy=Nix"
+  ];
+
+  preBuild = ''
+    go generate ./...
+  '';
+
+  postBuild = ''
+    PATH=$PATH:$GOPATH/bin
+    cunicu docs --with-frontmatter
+  '';
+
+  postInstall = ''
+    install -d $out/usr/share/man/man1
+    install ./docs/usage/man/*.1 $out/usr/share/man/man1
+    install -D <(cunicu completion bash) $out/share/bash-completion/completions/cunicu
+    install -D <(cunicu completion fish) $out/share/fish/vendor_completions.d/cunicu.fish
+    install -D <(cunicu completion zsh) $out/share/zsh/vendor-completions/_cunicu
+  '';
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
    cunicu docs --with-frontmatter
    installManpage ./docs/usage/man/*.1
    installShellCompletion \
      --bash <(cunicu completion bash) \
      --zsh <(cunicu completion zsh) \
      --fish <(cunicu completion fish)
  '';
```

https://nixos.org/manual/nixpkgs/unstable/#installshellfiles

---

Fixed in 295d96d041d31a0301f5f7b5599a232e5a537f3c